docs/README.md

```diff
@@ -1,6 +1,11 @@
-# Duke
+# Upcycle
 
-{Give product intro here}
+Hi there, 
+
+Welcome to Upcycle - a perfect desktop app dedicate for managing rental businesses. The nature of these businesses
+requires to deal with tons of data; therefore, noting down all information by notes and pen is not ideal for managers.
+It is developed for rental business managers, who can type fast to efficiently keep track all of their customers, items,
+and transactions via Command Line Interface with a single line of command.
 
 Useful links:
 * [User Guide](UserGuide.md)
```

Comment:
Hi this 0 can be omitted~ But it's optional just want u to know
![5](https://user-images.githubusercontent.com/71815678/198497303-3ce4fac6-5ad9-4f92-977f-f707613fbd92.png)